libr/core/cmd.c

```diff
@@ -1272,6 +1277,23 @@ static int cmd_ls(void *data, const char *input) { // "ls"
 			eprintf ("Usage: less [file]\n");
 		}
 		break;
+	case 'r':
+		files = r_list_new ();
```

Comment:
better move this logic into cmd_lsr(RCore *core, const char *input); so you can put those local variables locally there instead of having more variables that are only used in one execution path in this function.